osmaxx-py/excerptconverter/converter_helper.py

```diff
@@ -7,6 +6,50 @@
 from osmaxx.excerptexport import models
 
 
+def module_converter_configuration(name, export_formats, export_options):
+    return {
+        'name': name,
+        'formats': export_formats,
+        'options': export_options
+    }
+
+
+def run_model_execute(execute_task_method, export_formats,
+                      extraction_order, execution_configuration, run_as_celery_tasks):
```

Comment:
Avoid positional boolean arguments. (See 'Clean Code', Chapter 17, Section 'Functions', **F3: Flag Arguments** and Section 'General', **G15: Selector Arguments**.) Splitting the function into two separate ones as 'Clean Code' recommends is not practical here, though.

Thus, can we use a keyword argument instead, with `True` as default value? That way, only the invocation in some tests will be cluttered, and the "clutter" will help understand the meaning of this flag.